doc/gpo/assets/PowerToys.admx

```diff
@@ -0,0 +1,276 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft Corporation.
+     Licensed under the MIT License. -->
+<policyDefinitions xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" revision="1.0" schemaVersion="1.0" xmlns="http://schemas.microsoft.com/GroupPolicy/2006/07/PolicyDefinitions">
+  <policyNamespaces>
+    <target prefix="powertoys" namespace="Microsoft.Policies.PowerToys" />
+    <using prefix="windows" namespace="Microsoft.Policies.Windows" />
+  </policyNamespaces>
+  <resources minRequiredRevision="1.0" />
+  <categories>
+    <category name="PowerToys" displayName="$(string.PowerToys)">
+    </category>
+  </categories>
+  <policies>
+    <policy name="ConfigureEnabledUtilityAlwaysOnTop" class="Both" displayName="$(string.ConfigureEnabledUtilityAlwaysOnTop)" explainText="$(string.ConfigureEnabledUtilityDescription)" key="Software\Policies\PowerToys" valueName="ConfigureEnabledUtilityAlwaysOnTop">
+      <parentCategory ref="PowerToys" />
+      <supportedOn ref="windows:SUPPORTED_Windows_10_0_20H1" />
```

Comment:
This is not what I mean. If you install an older PT version on the newest Windows version the policy won't work anyway.

Here we have to create our own strings like "At least PowerToys v0.64.0". (And later on special cases something like "At least PowerToys v0.64.0 on Windows 11". - E.g. for some W11-only FZ settings.)

**Example from Firefox ADMX:**
https://github.com/mozilla/policy-templates/blob/684c1a72a17d3785d4f1fde3b49d8486fe0de93a/windows/firefox.admx#L11
https://github.com/mozilla/policy-templates/blob/684c1a72a17d3785d4f1fde3b49d8486fe0de93a/windows/firefox.admx#L149
https://github.com/mozilla/policy-templates/blob/684c1a72a17d3785d4f1fde3b49d8486fe0de93a/windows/en-US/firefox.adml#L9

Reply:
I've done this. Thanks for the tip.